rero_ils/modules/items/models.py

```diff
@@ -92,3 +92,10 @@ class ItemNoteTypes:
         CONDITION,
         PATRIMONIAL
     ]
+
+    INVENTORY_LIST = [
```

Comment:
Can you change the name such as `INVENTORY_LIST_CATEGORY` or similar?

Reply:
ok